src/node.cc

```diff
@@ -3535,6 +3547,7 @@ static void PrintHelp() {
          "  --trace-deprecation        show stack traces on deprecations\n"
          "  --throw-deprecation        throw an exception on deprecations\n"
          "  --no-warnings              silence all process warnings\n"
+         "  --napi-modules[=yes|no]    load N-API modules (default no)\n"
```

Comment:
I agree. Once non-experimental we would would most likely just remove the flag.  I think at this point we could just likely make it --napi-modules with no options.